### PR TITLE
text/javascript" is used (should be just <script>)

### PR DESCRIPTION
**Pull request by adeelq**

### CODE REVIEW

File: view/frontend/templates/email/email_capture_checkout.phtml [hunk 1]
```diff
@@ -2,7 +2,7 @@
```
Comment (cdiacon) [inline view/frontend/templates/email/email_capture_checkout.phtml]: you can't use $block to get the helper from template - use $this

File: view/frontend/templates/email/tracking_code.phtml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?php $helper = $block->helper('Dotdigitalgroup\Email\Helper\Data'); ?>
```
Comment (cdiacon) [inline view/frontend/templates/email/tracking_code.phtml]: you can't use $block to get the helper from template - use $this

File: view/frontend/templates/mailcheck/init.phtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?php if($block->helper('Dotdigitalgroup\Email\Helper\Data')->isMailCheckEnabledForCurrentStore()): ?>
-    <script type="text/javascript">
+    <script>
```
Comment (cdiacon) [inline view/frontend/templates/mailcheck/init.phtml]: you can't use $block to get the helper from template - use $this

**Comment by adeelq** [comment]
@cdiacon all above is already done in this pull request https://github.com/dotmailer/dotmailer-magento2-extension/pull/308/files

**Pull request merged**